src/include/plan_node/limit_plan_node.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+#include "catalog/schema.h"
+#include "plan_node/abstract_plan_node.h"
+
+// TODO(Gus,Wen): I don't think limit really needs an output schema. I'll include it for now, but we can maybe toss it
```

Comment:
I thought we decided that every PlanNode should have one?

Reply:
Limit will never change the projection from its child plan, but for consistency sake we can include the output schema. I've deleted the comment